dev/ProgressRing/ProgressRing.xaml

```diff
@@ -0,0 +1,75 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <Style TargetType="local:ProgressRing">
+        <Setter Property="Foreground" Value="{ThemeResource SystemControlHighlightAccentBrush}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource SystemControlBackgroundBaseLowBrush}" />
+        <Setter Property="StrokeThickness" Value="4" />
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:ProgressRing">
+                    <Grid x:Name="LayoutRoot" Background="{TemplateBinding Background}">
+                        <!-- Gray outline of progress bar -->
+                        <Path
+                            Fill="{TemplateBinding Background}"
+                            Stroke="{TemplateBinding BorderBrush}"
+                            StrokeThickness="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=StrokeThickness}"
```

Comment:
Does `StrokeThickness="{TemplateBinding StrokeThickness}"`  work?

Reply:
Yes, the relative binding was for when it was `BorderProperty`. Reverted.